hr_employee_department_secondary/views/hr_leave_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="hr_view_leave_form_multidepartment">
+        <field name="name">hr.leave.form.inherit</field>
+        <field name="model">hr.leave</field>
```

Comment:
i think you can create another module in `hr-holidays` repository and depends on this module.